core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1369,7 +1377,10 @@ class ReplicaManager(val config: KafkaConfig,
               val requestLeaderEpoch = partitionState.leaderEpoch
               val requestTopicId = topicIds.get(topicPartition.topic)
```

Comment:
Could we simplify this by taking advantage of `getTopicId`?

Reply:
Ah, this will simplify `hasConsistentTopicId`